resolvers/webpack/index.js

```diff
@@ -44,12 +44,16 @@ exports.resolve = function (source, file, settings) {
 
   var webpackConfig
 
-  var configPath = get(settings, 'config')
+  var _configPath = get(settings, 'config')
     , configIndex = get(settings, 'config-index')
     , env = get(settings, 'env')
     , argv = get(settings, 'argv', {})
     , packageDir
 
+  var configPath = typeof _configPath === 'string' && _configPath.startsWith('./')
```

Comment:
a relative path starts with any `.`
```suggestion
  var configPath = typeof _configPath === 'string' && _configPath.startsWith('.')
```